Získávání veškerých informací o verzích Ubuntu skrze API

MikkCZ opened the pull request:
Pokusil jsem se v Ruby napsat stahování informací o verzích včetně toho, které jsou poslední, ze dvou JSONů od Canonicalu.

Některé starší verze nemají kompletní data, nebo se stále zobrazují jako podporované, i když nejsou. Obešel jsem to tak, že se zobrazuje jen asi posledních 10 verzí.

fix #24

MikkCZ committed:
Získávání veškerých informací o verzích Ubuntu skrze API, #24

MikkCZ committed:
Oprava seznamu historie verzí

zubozrout reviewed (commented):
To vypadá super :). Zkusil jsem to a funguje. A na žádný problém jsem nenarazil.

Jen možná bys mohl přidat do o-ubuntu/historie-vydani i číslo verze - v api: "version", třeba do závorky za název. Nebylo tam, ale když nad tím tak přemýšlím, tak málo kdo si asi pamatuje Ubuntu názvy, zatímco čísla jsou docela jasná.
Tj, pak by tam mohlo být např.: "Focal Fossa (20.04)".

MikkCZ committed:
Vylepšení seznamu historie verzí

- využití volného místa
- zobrazení čísla verze
- datum vydání a ukončení podpory pro všechny verze

MikkCZ commented:
Dobrý nápad. Číslo verze jsem přidal a přidal jsem i datum vydání a ukončení podpory pro všechny verze v seznamu. Vypadá to teď takhle:

![obrazek](https://user-images.githubusercontent.com/3941344/68533411-e2fc9280-0328-11ea-9b2e-35d64ed1e5d4.png)

zubozrout commented:
To je super, děkuji @MikkCZ :).

zubozrout commented:
A u mě to funguje hezky :+1:, takže bych řekl, že je to ready to go...
